.circleci/config.yml

```diff
@@ -37,9 +37,15 @@ jobs:
       name: default
     steps:
       - checkout
+      - run: |
```

Comment:
`*restore_cache` and `*save_cache` should wrap any `go` commands -- this speeds up the builds by allowing CircleCI to leverage the go module store from previous invocation builds.

Reply:
@lieut-data replace the order